.github/workflows/prepare-for-minor-release.yml

```diff
@@ -48,7 +48,7 @@ jobs:
         with:
           title: Prepare repository for new minor release branch ${{ inputs.release_branch_name }}
           commit-message: Prepare repository for new minor release branch ${{ inputs.release_branch_name }}
-          branch: ${{ github.event.client_payload.release_branch_name }}-prepare-for-minor-release
+          branch: prepare-${{ github.event.client_payload.release_branch_name }}-for-minor-release
           base: ${{ github.event.client_payload.release_branch_name }}
```

Comment:
Not sure if we need this branch name to correspond to one on az-digital/az_quickstart.

Reply:
<img width="1596" alt="image" src="https://github.com/az-digital/az-quickstart-scaffolding/assets/1023167/5fb799fe-f313-4c33-bc67-55db087cb06b">